deployments/puppet/README.md

```diff
@@ -42,11 +42,15 @@ class accepts the following parameters:
  - `$config_file_path`: The path of the config file that will be rendered by the
    module (**default:** '/etc/signalfx/agent.yaml')
 
- - `$version`: The agent package version.  This is of the form `<agent
-	 version>-<package revision>` (e.g. package version `3.0.1-1` is the first
-	 package revision that contains the agent version `3.0.1`).  Releases with
-	 package revision > 1 contain changes to some aspect of the packaging
-	 scripts (e.g. init scripts) but contain the same agent bundle.
+ - `$agent_version`: The agent release version, in the form `1.1.1`.  This
```

Comment:
Clarify which one of these takes priority if both get set.

Reply:
Added a note under `package_version` option definition